pp_rework/pp_example.xml

```diff
@@ -0,0 +1,259 @@
+    <Function class="Functions" type="External">goalFunctionForLimitSurface</Function>
+  </PostProcessor>
+</Models>
+
+<PostProcessors>
+  <LimitSurface name="computeLimitSurfacePositiveNegative">
+    <parameters>x0,y0</parameters>
+    <side>both</side>
+    <ROM class="Models" type="ROM">Acc</ROM>
+    <!--You can add here a ROM defined in Models block.If not Present, a nearest algorithm is going to be used-->
+    <Function class="Functions" type="External">goalFunctionForLimitSurface</Function>
+  </LimitSurface>
+</PostProcessors>
+
+<!-- Metrics -->
+<Models>
+  <PostProcessor name="pp1" subType="Metric">
+    <Features type="variable">ans</Features>
+    <Targets type="variable">ans2</Targets>
+    <Metric class="Metrics" type="SKL">euclidean</Metric>
+    <Metric class="Metrics" type="SKL">euclidean2</Metric>
+    <Metric class="Metrics" type="SKL">cosine</Metric>
+    <Metric class="Metrics" type="SKL">manhattan</Metric>
+  </PostProcessor>
+</Models>
+
+<PostProcessors>
+  <EnsembleMerics name="pp1">
+    <Features type="variable">ans</Features>
+    <Targets type="variable">ans2</Targets>
+    <Metric class="Metrics" type="SKL">euclidean</Metric>
+    <Metric class="Metrics" type="SKL">euclidean2</Metric>
+    <Metric class="Metrics" type="SKL">cosine</Metric>
+    <Metric class="Metrics" type="SKL">manhattan</Metric>
+  </EnsembleMetrics>
+</PostProcessors>
+
+<!-- SampleSelector -->
+<Models>
+  <PostProcessor name="PPmax" subType="SampleSelector" verbosity="debug">
+    <target> y1 </target>
+    <criterion> max </criterion>
+  </PostProcessor>
+</Models>
+
+<PostProcessors>
+  <SampleSelector name="PPmax">
+    <target> y1 </target>
+    <criterion> max </criterion>
+  </SampleSelector>
+</PostProcessors>
+
+<!-- TopologicalDecomposition -->
+<Models>
+  <!-- Nearest neighbor -->
+  <PostProcessor name="AMSC_NN" subType="TopologicalDecomposition">
+    <graph>Approximate KNN</graph>
+    <gradient>steepest</gradient>
+    <knn>9</knn>
+    <normalization>None</normalization>
+    <parameters>X,Y</parameters>
+    <response>Z</response>
+    <simplification>0.2</simplification>
+    <persistence>difference</persistence>
+  </PostProcessor>
+</Models>
+<!-- More work need to be done!!! -->
+<PostProcessors>
+  <TopologicalDecomposition name="AMSC_NN">
+    <graph>Approximate KNN</graph>
+    <gradient>steepest</gradient>
+    <knn>9</knn>
+    <normalization>None</normalization>
+    <parameters>X,Y</parameters>
+    <response>Z</response>
+    <simplification>0.2</simplification>
+    <persistence>difference</persistence>
+  </TopologicalDecomposition>
+</PostProcessors>
+
+<!-- ValueDuration -->
+<Models>
+  <PostProcessor name="val_dur" subType="ValueDuration">
+    <target>Speed</target>
+    <bins>100</bins>
+  </PostProcessor>
+</Models>
+
+<PostProcessors>
+  <ValueDuration name="val_dur">
+    <target>Speed</target>
+    <bins>100</bins>
+  </ValueDuration>
+</PostProcessors>
```

Comment:
I really like how the PostProcessors blocks look. The reduced abstraction layer makes it easier to find the information that matters, I think.